packages/radio-player/src/search-results-switcher.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  CSSResult,
+  TemplateResult,
+  PropertyValues,
+} from 'lit-element';
+
+import leftImage from './assets/img/left';
+import rightImage from './assets/img/right';
+
+@customElement('search-results-switcher')
+export default class SearchResultsSwitcher extends LitElement {
+  @property({ type: Number }) numberOfResults = 0;
+
+  @property({ type: Number }) currentResultIndex = 0;
+
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <a @click=${this.goToPreviousResult} id="previous-button">${leftImage}</a>
```

Comment:
suggestion: use a `button` since it is not a link or page anchor.
is `id` absolutely necessary?
suggestion: wrap `svg` in image tag to define what the image is used for

Reply:
I switched it to use a `button` instead of a link. I need the `id` there to test clicking it. The image now exists inside the button so I hope it's more clear that it's the button's image.